sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StreamingWriteFn.java

```diff
@@ -41,11 +41,12 @@
  */
 @SystemDoFnInternal
 @VisibleForTesting
-class StreamingWriteFn
+class StreamingWriteFn<T>
```

Comment:
Rename T to be ErrorT, or something more descriptive here. T is often used for element type, so this is confusing.

Reply:
Done in f2c0460